README.md

```diff
@@ -33,6 +33,18 @@ CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.DYNAMODB_ADAPTER AS
 /
 ```
 
+In addition to the adapter script we must create an UDF function that will handle the loading of the data:
+```
+CREATE OR REPLACE JAVA SET SCRIPT ADAPTER.ImportDocumentData(
```

Comment:
Tru to keep names in all examples unified. Here you have schema name in uppercase and script name in camel-case. It would be better to choose one style and use it everywhere.